### ISSUE TITLE
Remove warnings on compilation

### ISSUE THREAD

**Issue by AndrewDryga** [igas/faker#60]
I guess it would be nice to remove all warnings from a console:

```
warning: crypto:rand_bytes/1 is deprecated and will be removed in a future release; use crypto:strong_rand_bytes/1
  lib/faker/bitcoin.ex:13

warning: crypto:rand_bytes/1 is deprecated and will be removed in a future release; use crypto:strong_rand_bytes/1
  lib/faker/bitcoin.ex:17

warning: random:seed/1: the 'random' module is deprecated; use the 'rand' module instead
  lib/faker/code.ex:5

warning: random:seed/1: the 'random' module is deprecated; use the 'rand' module instead
  lib/faker/code.ex:11

warning: random:seed/1: the 'random' module is deprecated; use the 'rand' module instead
  lib/faker/code.ex:17

warning: random:seed/1: the 'random' module is deprecated; use the 'rand' module instead
  lib/faker/avatar.ex:43

warning: random:seed/1: the 'random' module is deprecated; use the 'rand' module instead
  lib/faker/avatar.ex:48

warning: random:seed/1: the 'random' module is deprecated; use the 'rand' module instead
  lib/faker/internet.ex:31

warning: random:seed/1: the 'random' module is deprecated; use the 'rand' module instead
  lib/faker/internet.ex:61

warning: random:seed/1: the 'random' module is deprecated; use the 'rand' module instead
  lib/faker/address.ex:60

warning: random:uniform/0: the 'random' module is deprecated; use the 'rand' module instead
  lib/faker/address.ex:61

warning: random:seed/1: the 'random' module is deprecated; use the 'rand' module instead
  lib/faker/address.ex:69

warning: random:uniform/0: the 'random' module is deprecated; use the 'rand' module instead
  lib/faker/address.ex:70
```

**Comment by igas** [comment]
Hi, can you update faker. I think it was fixed in #53 and 0.7.0 were released ~ month ago. If you still get warnings pls tell me.

**Comment by igas** [comment]
I suppose update helped, closing this one.
